src/mattsum/boot_react_native.clj

```diff
@@ -57,18 +57,19 @@
 
 (deftask replace-main
   "Replaces the main.js with a file that can be read by React Native's packager"
-  [o output-dir OUT str  "The cljs :output-dir"]
+  [o output-dir OUT str  "The cljs :output-dir"
+   _ output-to OUT str "The cljs :output-to"]
   (let []
     (with-pre-wrap fileset
       (let [out-dir (str (or output-dir "main.out") "/")
             tmp (c/tmp-dir!)
-            main-file (->> "main.js"
+            main-file (->> (or output-to "main.js")
```

Comment:
Output to might be a path (e.g. `rn/main.js`) and that should work correctly with `find-file`.